### ISSUE TITLE
[Accessibility reflow issue] - Hamburger navigation menu has a min width of 350px causing a horizontal bar for smaller screens (MAS issue: 259081a)

### ISSUE THREAD

**Issue by ckbedwell** [grafana/grafana#72881]
**Steps to reproduce**:
Go to any page with the viewport at 320px width. (Or a common screen size of 1280px with the zoom set at 400%). Open up the hamburger menu. See have to scroll side to side to read all of the content.

**Actual Result**:
See content cut off and need to scroll to read it all.

**Expected Result**
There should be no need to scroll. The content should reflow using the vertical space.

![image](https://github.com/grafana/grafana/assets/6906380/f1a45d4c-7179-493d-92e4-8236c2f3bf3e)

**Comment by ckbedwell** [comment]
https://github.com/grafana/grafana/issues/77411 Very similar but ever-so-slightly different issue. Linking to each other to ensure they are both fixed when this is tackled.

**Comment by ashharrison90** [comment]
this is also fixed 👍
